### DISCUSSION TITLE
第3节课作业(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#191] [作业]
第3节课作业(13班)👇

**Comment by qingyin741** [comment]
复现课程知识库助手搭建过程 (截图)：
![Snipaste_2024-01-09_10-39-27](https://github.com/InternLM/tutorial/assets/118804949/ae01212f-edcd-4e8b-9340-305366ac01e4)
![Snipaste_2024-01-09_11-27-42](https://github.com/InternLM/tutorial/assets/118804949/9da1e413-d962-414b-8d82-0395c9cb5889)

![Snipaste_2024-01-09_12-24-29](https://github.com/InternLM/tutorial/assets/118804949/ec2d7451-d1f1-400c-8105-823fbf928a76)
![Snipaste_2024-01-09_12-25-59](https://github.com/InternLM/tutorial/assets/118804949/4106c72f-80c8-419d-a2c9-b23aa43d7d2b)
(⊙﹏⊙)说实话，有一点点人工智障

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/PP5GwoEVsiLUZtkLmjFc5Ig2nsh?from=from_copylink

**Comment by SiliconSisyphus** [comment]
前期环境的搭建
<img width="907" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/4e25845c-4abb-40d8-9618-45f6f7de20ee">
后期生成的对话界面
<img width="1082" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/490bd482-a2ae-4b43-85b8-e7833616b10f">

**Reply by SiliconSisyphus** [reply]
补发一条，感觉回答怪怪的
<img width="925" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/4135c8fb-e86f-4c96-9664-9b4456d81954">

**Comment by utopia2077** [comment]
https://nibqzjipp3p.feishu.cn/docx/SkPedmcckoVPCXxXu3IcoJeFn4L?from=from_copylink

**Comment by zhaoyiz0606** [comment]
<img width="1259" alt="image" src="https://github.com/InternLM/tutorial/assets/55607803/a1d8ef68-4f35-4134-92ae-47d3ba2364f0">

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135560537

**Comment by qingtao12138** [comment]
https://docs.qq.com/doc/DREVQWXV5anBzZG5q

**Comment by Taixi-Feng** [comment]


**Reply by Lum1104** [reply]
这个是啥

**Reply by Taixi-Feng** [reply]
![37a6e1e879b13b81a24b7f1eddd58ad](https://github.com/InternLM/tutorial/assets/104333661/b6c0c783-2227-4056-9ee8-f1f738422b57)

**Comment by bianchengyuanyuan** [comment]
https://ou3v176g8k.feishu.cn/docx/WkvmdlQu9oqAqqxROYNcxwOTnsh

**Comment by 5Fbw** [comment]
作业3
https://fifth-tower-f54.notion.site/493403d7193c4380894a037363c83bbb?pvs=4

**Comment by RessCris2** [comment]
https://blog.csdn.net/weixin_41783424/article/details/135588332?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135588332%22%2C%22source%22%3A%22weixin_41783424%22%7D

**Comment by suwief** [comment]
https://zhuanlan.zhihu.com/p/678185198
